R/rlang-options.R

```diff
@@ -0,0 +1,20 @@
+#' Global options for rlang
+#'
+#' @description
+#' rlang has several options which may be set globally to control
+#'  behavior. A brief description of each is given here. If any functions
+#'  are referenced, refer to their documentation for additional details.
+#'
+#' * rlang_interactive: A logical value used by [is_interactive()]. This
+#'    can be set to `TRUE` to test interactive behavior in unit tests,
+#'    for example.
+#' * rlang_backtrace_on_error: A character string which controls whether
+#'    backtraces are displayed with error messages, and the level of
+#'    detail they print. See [rlang_backtrace_on_error] for the possible option values.
+#' * rlang_trace_format_srcrefs: A logical value used to control whether
+#'    srcrefs are printed as part of the backtrace.
+#' * rlang_trace_top_env: An environment which will be treated as the
+#'    top-level environment when printing traces. See [trace_back()]
+#'    for examples.
+#' @name rlang_options
```

Comment:
I think the topic could be called `faq-options`, this will ensure it's included in the pkgdown reference along other faq items.

Reply:
I've left the file as `rlang-options.R`, but am happy to change that as well.

Reply:
Thanks, `faq.R` might be appropriate.

Reply:
The rlang pkgdown does not have a FAQ section, though it does have  "session state", which includes `matches("option")`. I had thought that had been the section you were referring to. 

Would you like me to update the pkgdown so it does have a FAQ section, similar to what [tidyselect](https://tidyselect.r-lib.org/reference/index.html#section-faq) has?

Reply:
That'd be great, thanks!